src/views/Profile/index.tsx

```diff
@@ -63,10 +63,10 @@ const Profile: React.FC<{ isLoading?: boolean }> = props => {
 
     const signAndUpdate = async (data: any) => {
         try {
-            const messageToSign = `I prove my identity by signing this. Current timestamp is ${timestamp}`;
+            const messageToSign = `${config.signatureMessage} ${timestamp}`;
 
             await signMessage(messageToSign)
-                .then((signature) => {
+                .then((signature: string) => {
                     registerSignature(signature, messageToSign);
             })
```

Comment:
IMHO, we should move to a separate file in order to use it with other forms.

Reply:
Created the handleSignature helper to handle the sign process and store its results on the cookies.